docs/notebooks/meta_learners_synthetic_data.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>I think we are missing a conclusion (and plots) regarding which meta-learners recover the CATE estimations better (and why).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>